packages/babel-plugin-transform-classes/src/transformClass.js

```diff
@@ -19,22 +19,6 @@ function buildConstructor(classRef, constructorBody, node) {
   return func;
 }
 
-const verifyConstructorVisitor = traverse.visitors.merge([
-  environmentVisitor,
-  {
-    Super(path, state) {
-      if (state.isDerived) return;
-
-      const { node, parentPath } = path;
-      if (parentPath.isCallExpression({ callee: node })) {
-        throw path.buildCodeFrameError(
-          "super() is only allowed in a derived constructor",
-        );
-      }
-    },
-  },
-]);
-
```

Comment:
Not sure if i should leave that or not. If this check would be in traverse I would leave it, but in the transform, the only valid usecase now would be someone creating in a plugin invalid code and then running the transform.

Reply:
I don't think that we need it: currently all the other early errors (except for the duplicated bindings errors) are only in `@babel/parser`.